pkg/virt-controller/watch/vm_test.go

```diff
@@ -3104,6 +3104,21 @@ var _ = Describe("VirtualMachine", func() {
 			Expect(vmi.Annotations).To(Equal(annotations))
 		})
 
+		It("should not copy annotations from vm to vmi", func() {
+			vm, _ := DefaultVirtualMachine(true)
+			vm.Annotations = map[string]string{"kubevirt.io/test": "test"}
```

Comment:
```suggestion
			vm.Annotations["kubevirt.io/test"] = "test"
```
otherwise you overwrite some annotations that result in the controller returning prematurely

Reply:
Ah exactly. Good catch.